src/GNDStk/Node/src/write.hpp

```diff
@@ -66,10 +69,19 @@ std::ostream &write(std::ostream &os, const int level) const
 
 
 // -----------------------------------------------------------------------------
+// Helper
 // write(file name, int level)
 // For FileType::text
 // -----------------------------------------------------------------------------
 
+private:
+
+// fixme I noticed that this write() variant isn't used (other write() functions
+// with filename go through ostream first, and call the earlier write() helper
+// with const int level). Decide if there's any reason to keep it. If we do keep
+// it, then it needs to be exercised in the test suite, which it isn't now.
+#if 0
```

Comment:
Sometimes I'll use an `#if 0`, as opposed to a block comment (or to removing the code entirely) basically as part of my own workflow process. I have various `grep` aliases that look for different types of markup that I may put into a code - block comments, fixmes, todos, etc. In all cases I'm trying to draw attention to something that I want to address at some point (but not at the moment, because I'm working on other things), and generally I'll only leave an `#if 0` sitting around if it's something I'm not sure I want to keep, or remove. In this particular instance, I have a fixme right above it, so I suppose I could just change it to a block comment if `#if 0` bothers anyone; either way, I'll find it. Speaking of the various `write()` functions, going through the entire code (which has grown considerably since I wrote the first `write()s`), and ensuring that the various flavors of `write()` are consistent with one another and reasonably clear to potential users, is relatively high on my priority list. So, I'll probably be looking at this exact code sometime soon. At that point, I'll either put the function in question back in, or remove it entirely.